cores/cv32e40p/user_manual/source/control_status_registers.rst

```diff
@@ -30,7 +30,17 @@ any overhead that we do not explicitly need.
 +-------------------+-----------+------------+------------+---------------+---------------+-------+--------------------------------------------------------+
 | 01                | 11        | 11         | 010010     | 0x7D2         | MIP1          | R     | Machine Interrupt Pending 1 Register                   |
 +-------------------+-----------+------------+------------+---------------+---------------+-------+--------------------------------------------------------+
-| 01                | 11        | 10         | 110xxx     | 0x7B0-0x7B7   | HWLP          | R/W   | Hardware Loop Registers                                |
+| 01                | 11        | 10         | 110000     | 0x7C0         | LPSTART[0]    | R/W   | Hardware Loop 0 Start                                  |
```

Comment:
Can you use the actual CSR names (as used in the assembly), so e.g. LPSTART0 instead of LPSTART[0]? Same for the other CSRs